src/main/java/TTT/controller/CustomUserController.java

```diff
@@ -0,0 +1,34 @@
+package TTT.controller;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class CustomUserController {
+
+
+    @GetMapping("/user")
+    public String getMainPage() {
+        return "user";
+    }
+
+    @GetMapping("/map")
+    public String getMapPage() {
+        return "map";
+    }
+
+    @GetMapping("/announcement")
+    public String getAnnouncementsPage() {
+        return "announcement";
+    }
```

Comment:
To be moved from this controller

Reply:
Done. Created new controller Class to announcements.